src/plugins/data/common/index_patterns/types.ts

```diff
@@ -184,6 +185,11 @@ export interface FieldSpec {
 
 export type IndexPatternFieldMap = Record<string, FieldSpec>;
 
+export interface SavedObjectReference {
```

Comment:
@kristenTian Can you take a look?

Reply:
Sure. --  As for this one, the repo currently forbids import from` core `into` common `path. Also as you point out, the name could be optional here, hence I can rename the interface name here to avoid confusion.